src/nav/apm.yml

```diff
@@ -1310,10 +1310,8 @@ pages:
             path: /docs/apm/transactions/transaction-traces/introduction-transaction-traces
           - title: Configure transaction traces
             path: /docs/apm/transactions/transaction-traces/configure-transaction-traces
-          - title: Summary page
-            path: /docs/apm/transactions/transaction-traces/transaction-traces-summary-page
           - title: Trace details page
-            path: /docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page
+            path: /docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page/
```

Comment:
I've seen in the files this other path: `/docs/apm/transactions/transaction-traces/transaction-trace-details-page` and here I see a different one. Maybe is OK, but, just in case you want to check it out ;-)

Reply:
I think I fixed this!